app/views/transactions/filter.njk

```diff
@@ -209,6 +209,37 @@
         </fieldset>
       </fieldset>
     </div>
+    {% if isExperimentalFeaturesEnabled %}
+      <div class="govuk-grid-column-full">
+        <div class="govuk-grid-column-one-half">
+          <details class="govuk-details" data-module="govuk-details">
+            <summary class="govuk-details__summary">
+              <span class="govuk-details__summary-text">
+                Advanced filters
+              </span>
+            </summary>
+            <div class="govuk-details__text">
```

Comment:
Detail is currently not expanded when search is performed on metadata field. I am assuming this is expected and to be added in a separate PR ?

Reply:
Good spot - planning to do that in another PR.
Thanks.